src/modules/Radio/Radio.tsx

```diff
@@ -0,0 +1,123 @@
+import React from "react";
+import uuid from "uuid/v4";
+import {
+  StandardProps,
+  WithStylesProps,
+  ThemeStyleSheetFactory
+} from "../../types";
+import { withStyles } from "../../base";
+
+interface IProps extends StandardProps<"input"> {
+  hasError?: boolean;
+  isDisabled?: boolean;
+  label: string;
+  name?: string;
+  selectedValue?: string;
+  value: string;
```

Comment:
same as before

Reply:
We need to force the type bc input's "value" can be undefined :(